pkg/graveler/graveler.go

```diff
@@ -780,16 +784,20 @@ func (id TagID) String() string {
 }
 
 type KVGraveler struct {
-	db      *DBGraveler
-	kvStore *kv.StoreMessage
-	log     logging.Logger
+	db         *DBGraveler
+	RefManager RefManager
+	hooks      HooksHandler
+	kvStore    *kv.StoreMessage
```

Comment:
You can remove the kvStore from graveler. We don't really need it here

Reply:
done